core/elements/ons-icon.es6

```diff
@@ -82,6 +82,7 @@ limitations under the License.
       var size = '' + this.getAttribute('size');
       if (size.match(/^[1-5]x|lg$/)) {
         classList.push('fa-' + size);
+        this.removeAttribute('style');
```

Comment:
Can you check that the previous value was "*px" and then just remove the `font-size` attribute?